satpy/readers/ahi_hsd.py

```diff
@@ -572,21 +636,25 @@ def read_band(self, key, info):
                               b=float(self.proj_info['earth_polar_radius'] * 1000))
         actual_alt = float(self.nav_info['distance_earth_center_to_satellite']) * 1000 - re
 
+        if self._round_actual_position:
+            actual_lon = round(actual_lon, 3)
+            actual_lat = round(actual_lat, 2)
+            actual_alt = round(actual_alt / 150) * 150  # to the nearest 150m
+
         # Update metadata
         new_info = dict(
-            units=info['units'],
-            standard_name=info['standard_name'],
-            wavelength=info['wavelength'],
+            units=ds_info['units'],
+            standard_name=ds_info['standard_name'],
+            wavelength=ds_info['wavelength'],
             resolution='resolution',
             id=key,
             name=key['name'],
-            scheduled_time=self.scheduled_time,
+            scheduled_start_time=self.scheduled_start_time,
+            scheduled_end_time=self.scheduled_end_time,
+            observation_start_time=self.observation_start_time,
+            observation_end_time=self.observation_end_time,
```

Comment:
Similarly to the orbital parameters, could we have a nested dict for times here?

Reply:
Yes, that could work. Do you have an idea for the name?

Reply:
Time_information?

Reply:
This PR or a separate one? I suppose it would have to be done in this one as we are defining the metadata. Shoot.

`time_parameters`?

Reply:
Yes, that works